foundation/foundation-mda/src/main/resources/templates/data-delivery-spark/asynchronous.processor.base.java.vm

```diff
@@ -26,7 +26,7 @@ import org.reactivestreams.Subscription;
 #if (${step.outbound})
 import org.eclipse.microprofile.reactive.messaging.Outgoing;
 #end
-#if ((${step.hasMessagingOutbound()} && ${step.hasNativeInbound()}))
+#if ((${step.hasMessagingOutbound()} && (${step.hasNativeInbound()} ||  ${step.hasVoidInbound()} )))
```

Comment:
I: I tried emitter instead of directly putting outgoing channel in method by referring this test [OutOnlyTestStep](https://github.com/boozallen/aissemble/blob/dev/extensions/extensions-data-delivery/extensions-data-delivery-spark/src/test/java/com/boozallen/aiops/data/delivery/messaging/pipeline/steps/OutOnlyTestStep.java) 


upon changing build seems to pass, also ran downstream project with problematic metamodel, seems to pass build as well as inspection of spark pipeline to make sure it succeeded. 


![Screenshot 2024-10-30 at 9 55 58 AM](https://github.com/user-attachments/assets/60d111fd-229d-446a-8f88-c73b7cc47035)
![Screenshot 2024-10-30 at 9 56 11 AM](https://github.com/user-attachments/assets/025bb570-813c-4091-9828-585b0cb1bc63)